.github/workflows/ansible-test-plugins.yml

```diff
@@ -282,46 +282,31 @@ jobs:
       fail-fast: true
       matrix:
         ansible:
-          - stable-2.16
           - stable-2.17
           - stable-2.18
+          - stable-2.19
           - devel
         python:
           - '3.8'
           - '3.9'
           - '3.10'
           - '3.11'
         exclude:
-          - python: '3.8'
-            ansible: stable-2.16
-
           - python: '3.8'
             ansible: stable-2.17
 
           - python: '3.8'
             ansible: devel
 
-          - python: '3.9'
-            ansible: stable-2.15
+          - python: '3.8'
+            ansible: stable-2.19
 
           - python: '3.9'
             ansible: stable-2.17
 
           - python: '3.9'
             ansible: devel
 
```

Comment:
```suggestion
          - python: '3.9'
            ansible: devel

          - python: '3.10'
            ansible: stable-2.17

          - python: '3.11'
            ansible: stable-2.17
```

Reply:
Without this, we would have two more tests suites than before.

Reply:
@laurent-indermuehle please commit your suggestion and merge if it's OK, my brain doesn't work well now:)